test/integration/Makefile

```diff
@@ -0,0 +1,30 @@
+BASE_PATH=$(shell dirname $BASH_SOURCE)
+
+# Finds the relative path of any modified or new CQL files for integration testing
+MODIFIED_CQL_FILES=$(shell git status --porcelain $(BASE_PATH)/**/*.cql | sed "s/^...//")
+
+# Prevents the presence of a directory with the same name skipping Makefile targets
+# TODO: Add other targets once defined
+.PHONY: all proportion-boolean
+
+# This loop grabs the important directory name from the full path to the cql file
+# This directory name MUST match the make target name in this file
+all: .start-translator
+	@for f in $(MODIFIED_CQL_FILES) ; do \
+		replacement_suffix=$${f%/cql/*}; \
+		make_target_name=$${replacement_suffix#test/integration/} ; \
+		make $$make_target_name ; \
+	done
+
+.start-translator:
+	docker pull cqframework/cql-translation-service:latest
+	docker run --name cql-translation-service --rm -dit -p 8080:8080 cqframework/cql-translation-service:latest
+	sleep 5
+	touch .start-translator
+
+proportion-boolean:
+	$(BASE_PATH)/proportion-boolean/generate.sh
```

Comment:
Open question from me:

Does it even make sense to have a `generate.sh` script in each subdirectory? The alternative is to put the ecqm-bundler command directly in this make target instead of having it call out to a script

Pros of current approach:

* Keeps Makefile clean (avoids long ecqm-bundler commands embedded in the mMkefile)

Cons of current approach:

* Requires a new file to be created in every directory (more steps)
* Adds some bloat

Pros of new approach:

* No need for additional generate.sh file

Cons of new approach:

* ecqm-bundler commands may get quite long and could be hard to read within the makefile

Reply:
The Makefile target would become this:


```makefile
proportion-boolean:
	ecqm-bundler -c ./proportion-boolean/cql/proportion-boolean.cql --deps ./helpers/cql/FHIRHelpers-4.0.1.cql --ipop ipp --numer numer --denom denom --scoring-code proportion --basis boolean -o ./proportion-boolean/proportion-boolean-bundle.json
```